lib/components/artist/__tests__/about-tests.js

```diff
@@ -0,0 +1,19 @@
+import 'react-native'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import About from '../about'
+
+jest.mock('../../opaque_image_view.js', () => 'AROpaqueImageView')
+
+it('renders properly', () => {
+  const artist = {
+    has_metadata: true,
+    is_display_auction_link: true,
+    articles: [],
```

Comment:
For now, yes. I'm covering every component with the most basic snapshot but eventually they should have more comprehensive tests. I figure a very basic foundation for all components is better than having several components that are extremely well-tested and others that are not tested at all.

There are just so many components and so little time.